### PR TITLE
Web1.4 Added code for publishing feedbacks

### PR DESCRIPTION
**Pull request by vukkesic**
- Worked on publishing functionality
- Get non pushed feedbacks 
- Pick feedback for pushing

### CODE REVIEW

**Review by stojke13 — APPROVED** [review]
Looks ok

File: WEB_Project/Repository/UserRepo/SecretaryRepo/SecretaryRepository.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace HealthClinic.Repository.UserRepo.SecretaryRepo
+{
+   public interface SecretaryRepository : GenericInterfaceCRUDDao<Secretary, int>
+   {
+        void SetSecretarysBusinessHours(List<Secretary> doctors, BusinessHoursModel businessHours);
+
+        void makeUpdateFor(Secretary entity);
+
+        List<Secretary> getAllFreeSecretaries(BusinessHoursModel businessHours);
```
Comment (stojke13) [inline WEB_Project/Repository/UserRepo/SecretaryRepo/SecretaryRepository.cs:20]: Methods names not capitalized

File: WEB_Project/Controllers/AppReviewController.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            this.appReviewService = new AppReviewService(dbContext);
+        }
+
+        [HttpGet]     //GET /api/appReview
+        public IActionResult GetAll()
+        {
+            List<AppReviewDTO> result = new List<AppReviewDTO>();
+            appReviewService.GetAllAppReviews().ForEach(appReview => result.Add(AppReviewAdapter.AppReviewToDto(appReview)));
+            return Ok(result);
```
Comment (stojke13) [inline WEB_Project/Controllers/AppReviewController.cs:32]: Prebaciti u servis

**Review by stojke13 — APPROVED** [review]
Solidan kod

**Review by milicatosic — APPROVED** [review]
Ispraviti naznačene konflikte

**Review by stojke13 — CHANGES_REQUESTED** [review]
**Neophodno resisti konflikte**

**Review by milicatosic — CHANGES_REQUESTED** [review]
Neophodno je rešiti naznačene konflikte

**Review by stojke13 — APPROVED** [review]
Conflicts solved , looks good

**Review by milicatosic — APPROVED** [review]
Conflicts resolved, all looks good.

**Pull request merged**